linebot/webhook.py

```diff
@@ -64,9 +95,12 @@ def validate(self, body, signature):
             hashlib.sha256
         ).digest()
 
-        return hmac.compare_digest(
-            signature.encode('utf-8'), base64.b64encode(gen_signature)
-        )
+        if hexversion >= 0x020707F0:
```

Comment:
Would you like to use `if hasattr(hmac, "compare_digest"):` ?
I think that it is readable.